addons/project/wizard/project_template_create_wizard.py

```diff
@@ -70,6 +71,19 @@ class ProjectTemplateRoleToUsersMap(models.TransientModel):
     _name = 'project.template.role.to.users.map'
     _description = 'Project role to users mapping'
 
+    def _domain_user_ids(self):
+        return [
+            ('share', '=', False),
+            ('active', '=', True),
+            ('project_role_ids', '=?', unquote('role_id if role_user_ids else False')),
```

Comment:
Coucou @odoo/rd-security 

Can I have an override? I used `unquote` to be able to define the dynamic part to correctly evaluate the fields in the right part of the condition of the domain used in the definition of `user_ids` field. It is safe since the content inside the unquote is just fields of the wizard and it is only used to get the domain for user_ids field. 🙂

Thanks in advance!

Have a nice afternoon! 🙂